packages/typescript-estree/src/create-program/createProjectService.ts

```diff
@@ -0,0 +1,59 @@
+import type * as tsserverType from 'typescript/lib/tsserverlibrary';
```

Comment:
FWIW you can probably just call this `ts`, unless you're trying to avoid potential confusion.